compiler/vm.nim

```diff
@@ -494,7 +494,7 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
     of opcLdObj:
       # a = b.c
       decodeBC(rkNode)
-      let src = regs[rb].node
+      let src = if regs[rb].kind == rkNodeAddr: regs[rb].nodeAddr[] else: regs[rb].node
```

Comment:
fixing the vmgen probably have other unwanted side effects.
at least this patch will have minimum side effects. lets call it emergency temporary patch until I completed my debugger to better visualize the vm.